### PR TITLE
Allow override of revealable of password

### PR DESCRIPTION
**Pull request by sophypal**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG

* Removed hardcoding of `revealable` for `password` input and use `renderer.options` as a pass through

### CODE REVIEW

File: addon/components/inputs/password.js [hunk 1]
```diff
@@ -25,6 +28,19 @@ export default AbstractInput.extend({
...
+  @readOnly
+  @computed('cellConfig')
+  options (cellConfig) {
+    const options = get(cellConfig, 'renderer.options')
+    return merge({revealable: true}, options)
+  },
+
+  @readOnly
+  @computed('options')
+  revealable (options) {
```
Comment (EWhite613) [inline addon/components/inputs/password.js]: Can alias `options.revealable`

Comment (sophypal) [reply addon/components/inputs/password.js]: Can do

Comment (gkchestertron) [reply addon/components/inputs/password.js]: agreed. 
```
...
import computed, {alias, readOnly} from 'ember-computed-decorators'
...
@readOnly
@alias('options.revealable') revealable,
```

**Comment by EWhite613** [comment]
Approved

**Pull request merged**